elixir/apps/domain/test/domain/events_test.exs

```diff
@@ -0,0 +1,3 @@
+defmodule Domain.EventsTest do
+  # TODO: Add integration tests to ensure side effects trigger broadcasts in general
```

Comment:
It this TODO meant to be filled in for this PR or is this coming in another PR?

Reply:
Another PR.